cma/purecma.py

```diff
@@ -179,19 +181,18 @@ def fmin(objective_fct, xstart, sigma,
 
 
 class CMAESParameters(object):
-    """static "internal" parameter setting for `CMAES`
-
-    """
+    """Static "internal" parameter setting for `CMAES`"""
     default_popsize = '4 + int(3 * log(N))'
     def __init__(self, N, popsize=None,
                  RecombinationWeights=None):
-        """set static, fixed "strategy" parameters once and for all.
+        """
```

Comment:
Is there a style guide which recommends a leading new line?

Reply:
I had certainly thought it did, but apparently not; either with or without is considered fine, although the core libraries tend to use with a leading new line IIRC.

Reply:
I'll go over the docstrings with reference to https://www.python.org/dev/peps/pep-0257/.

Reply:
One disadvantage of not using a leading new line is that one is supposed to fit a summary line as the first one (something I admittedly tend to forget), and it helps to make sure the line isn't so long that github won't display all of it.